src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorIntegrationTest.java

```diff
@@ -364,6 +361,42 @@ public void testDeNovo() {
         Assert.assertEquals(lociWithHighConfidenceDeNovo, new int[] {10130767, 10197999});
     }
 
+    @Test
+    public void addQdToGvsVcf() {
```

Comment:
add a comment explaining that this has QUALapprox but no QUAL and that is what this is testing

Reply:
done